scripts/qa_crowbarsetup.sh

```diff
@@ -2450,6 +2450,12 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        manila)
+            if [[ ! $(iscloudver 7plus) ]] ; then
```

Comment:
This was me who introduced complicated syntax :frowning: In order to keep your PR clean and focused maybe I should submit a separate one.